src/originalBudgetTrackingApp/transactions/dates.ts

```diff
@@ -0,0 +1,5 @@
+export const formatDate = (date: Date) => `${date.getDate()}/${date.getMonth()}/${date.getFullYear()} ${date.getHours()}:${date.getMinutes()}`;
+
+export const compareObjects = (a: { date: string }, b: { date: string }) => compareStrings(a.date, b.date);
+export const compareStrings = (a: string, b: string) => compare(new Date(a), new Date(b));
+export const compare = (a: Date, b: Date) => a.getTime() - b.getTime();
```

Comment:
compareStrings and compare are internal functions so I don't think they should be exported. And names could be clearer.


```suggestion
export const compareObjectsByDate = (a: { date: string }, b: { date: string }) => compareDateStrings(a.date, b.date);
const compareDateStrings = (a: string, b: string) => compareDates(new Date(a), new Date(b));
const compareDates = (a: Date, b: Date) => a.getTime() - b.getTime();
```